src/makeChange.js

```diff
@@ -1,5 +1,41 @@
 'use strict'
 
 export default function makeChange({price, amountGiven}) {
-  // your code here
+
```

Comment:
looks great , also the fact that you're using Math.floor as well.
Are the  console.logs necessary tough?